### PR TITLE
🌐 Add Ukrainian translation for `docs/uk/docs/tutorial/path-params.md`

### PR DESCRIPTION
**Pull request by ss-o-furda**
related: https://github.com/tiangolo/fastapi/issues/1748

### CODE REVIEW

**Comment by tiangolo** [comment]
📝 Docs preview for commit 8df3c84eb0ec176d0012e49e798f898ec764aef7 at: https://649a148205f6af108fd6d17d--fastapi.netlify.app

File: docs/uk/docs/tutorial/path-params.md [hunk 1]
```diff
@@ -0,0 +1,244 @@
...
+Ви можете оголосити тип параметра шляху у функції, використовуючи стандартні анотації типу Python:
+
+```Python hl_lines="7"
+{!../../../docs_src/path_params/tutorial002.py!}
+```
+
+У цьому випадку `item_id` оголошується як `int`.
+
+!!! check
+    Це надасть вам підтримку редактора всередині вашої функції, з перевіркою помилок, автозаповненням, тощо.
```
Comment (mezgoodle) [inline docs/uk/docs/tutorial/path-params.md:28]:
```suggestion
    Це надасть вам підтримку редактора всередині вашої функції, з перевіркою помилок, автозаповненням тощо.
```

File: docs/uk/docs/tutorial/path-params.md [hunk 2]
```diff
@@ -0,0 +1,244 @@
...
+    Це надзвичайно корисно під час розробки та налагодження коду, який взаємодіє з вашим API.
+
+## Документація
+
+А коли ви відкриєте браузер на <a href="http://127.0.0.1:8000/docs" class="external-link" target="_blank">http://127.0.0.1:8000/docs</a>, ви побачите автоматичну, інтерактивну документацію API, як-от:
+
+<img src="/img/tutorial/path-params/image01.png">
+
+!!! check
+    Знову ж таки, лише з тим самим оголошенням типу Python, **FastAPI** надає вам автоматичну інтерактивну документацію (інтегруючи інтерфейс користувача Swagger).
```
Comment (mezgoodle) [inline docs/uk/docs/tutorial/path-params.md:80]:
```suggestion
    Знову ж таки, лише з тим самим оголошенням типу Python, **FastAPI** надає вам автоматичну інтерактивну документацію (інтегруючи Swagger UI).
```

File: docs/uk/docs/tutorial/path-params.md [hunk 3]
```diff
@@ -0,0 +1,244 @@
...
+
+Потім створіть *параметр шляху* з анотацією типу, використовуючи створений вами клас (`ModelName`):
+
+```Python hl_lines="16"
+{!../../../docs_src/path_params/tutorial005.py!}
+```
+
+### Перевірте документацію
+
+Оскільки доступні значення для *параметра шляху* попередньо визначені, інтерактивна документація може добре показати їх:
```
Comment (mezgoodle) [inline docs/uk/docs/tutorial/path-params.md:150]:
```suggestion
Оскільки доступні значення для *параметра шляху* попередньо визначені, інтерактивна документація може показати їх:
```

File: docs/uk/docs/tutorial/path-params.md [hunk 4]
```diff
@@ -0,0 +1,244 @@
...
+
+Скажімо, у вас є *операція шляху* із шляхом `/files/{file_path}`.
+
+Але вам потрібно, щоб сам `file_path` містив *шлях*, наприклад, `home/johndoe/myfile.txt`.
+
+Отже, URL-адреса цього файлу буде схожна на: `/files/home/johndoe/myfile.txt`.
+
+### Підтримка OpenAPI
+
+OpenAPI не підтримує спосіб оголосити *параметр шляху*, щоб він містив всередині *шлях*, оскільки це може призвести до сценаріїв, які важко перевірити та визначити.
```
Comment (mezgoodle) [inline docs/uk/docs/tutorial/path-params.md:206]:
```suggestion
OpenAPI не підтримує спосіб оголошення *параметра шляху*, щоб він містив всередині *шлях*, оскільки це може призвести до сценаріїв, які важко перевірити та визначити.
```

File: docs/uk/docs/tutorial/path-params.md [hunk 5]
```diff
@@ -0,0 +1,244 @@
...
+    У цьому випадку URL-адреса буде: `/files//home/johndoe/myfile.txt`, з подвійною косою рискою (`//`) між `files` та `home`.
+
+## Підсумок
+
+З **FastAPI**, використовуючи короткі, інтуїтивно зрозумілі та стандартні оголошення типу Python, ви отримуєте:
+
+* Підтримка редактора: перевірка помилок, автозаповнення, тощо
+* "<abbr title="converting the string that comes from an HTTP request into Python data">Парсинг</abbr>" даних
+* Валідація даних
+* Анотація API та автоматична документація
```
Comment (mezgoodle) [inline docs/uk/docs/tutorial/path-params.md:240]:
```suggestion
* Підтримку редактора: перевірка помилок, автозаповнення, тощо
* "<abbr title="converting the string that comes from an HTTP request into Python data">Парсинг</abbr>" даних
* Валідацію даних
* Анотації API та автоматична документацію
```

File: docs/uk/docs/tutorial/path-params.md [hunk 6]
```diff
@@ -0,0 +1,244 @@
+# Параметри шляху
+
+Ви можете оголосити "параметри" шляху або "змінні" з тим самим синтаксисом, що використовується строками Python:
```
Comment (mezgoodle) [inline docs/uk/docs/tutorial/path-params.md:3]:
```suggestion
Ви можете оголосити "параметри" шляху або "змінні" з тим самим синтаксисом, що використовується стрічками Python:
```

File: docs/uk/docs/tutorial/path-params.md [hunk 7]
```diff
@@ -0,0 +1,244 @@
...
+
+Інакше шлях для `/users/{user_id}` відповідатиме також для `/users/me`, "думаючи", що він отримує параметр `user_id` зі значенням `"me"`.
+
+## Попередньо визначені значення
+
+Якщо у вас є *операція шляху*, яка отримує *параметр шляху*, але ви хочете, щоб можливі дійсні значення *параметру шляху* були попередньо визначені, ви можете використовувати стандартний Python <abbr title="Enumeration (перерахування)">`Enum` (`перерахування`)</abbr>.
+
+### Створіть клас `Enum`
+
+Імпортуйте `Enum` та створюйте підклас, який успадковується від `str` та від `Enum`.
```
Comment (romashevchenko) [inline docs/uk/docs/tutorial/path-params.md:124]:
```suggestion
Імпортуйте `Enum` та створіть підклас, який успадковується від `str` та від `Enum`.
```

File: docs/uk/docs/tutorial/path-params.md [hunk 8]
```diff
@@ -0,0 +1,244 @@
...
+
+### Перевірте документацію
+
+Оскільки доступні значення для *параметра шляху* попередньо визначені, інтерактивна документація може добре показати їх:
+
+<img src="/img/tutorial/path-params/image03.png">
+
+### Робота з *перерахуванням* у Python
+
+Значенням *параметра шляху* буде *членом перерахування*.
```
Comment (romashevchenko) [inline docs/uk/docs/tutorial/path-params.md:156]:
```suggestion
Значення *параметра шляху* буде *членом перерахування*.
```

**Review by romashevchenko — CHANGES_REQUESTED** [review]
Hi there! 
I'm Romana from DreamProIT. As a native speaker, I have reviewed the translation and made a few suggestions to improve it.

File: docs/uk/docs/tutorial/path-params.md (hunk 6 above)
Comment (rostik1410) [inline docs/uk/docs/tutorial/path-params.md:3]:
```suggestion
Ви можете оголосити "параметри" шляху або "змінні" з тим самим синтаксисом, що використовується рядками Python:
```

File: docs/uk/docs/tutorial/path-params.md (hunk 8 above)
Comment (rostik1410) [reply docs/uk/docs/tutorial/path-params.md:156]: @romashevchenko Тоді вже краще `членом переліку` або `членом перелічення`

**Pull request closed without merging**